Dockerfile

```diff
@@ -1,13 +1,20 @@
 # Stage 0, "build-stage", based on Node.js, to build and compile the frontend
-FROM node:10.22.0-alpine as build-stage
+FROM node:12-alpine as build-stage
```

Comment:
We need to keep an eye on stability after the upgrade

Reply:
We should be fine, Node is only used for building, in local we were already running Node 12 for development and backend.